extensions/arc/deployment/src/main/resources/dev-templates/tags/display-bean.html

```diff
@@ -1,7 +1,7 @@
-<span class="annotation" title="{it.scope.name}">@{it.scope.localName}</span><br/>
-{#for qual in it.qualifiers}
- {#if qual.name != 'javax.inject.Named' && qual.name != 'javax.enterprise.inject.Any' && qual.name != 'javax.enterprise.inject.Default'}
-   <span class="annotation" title="{qual.name}">@{qual.localName}</span><br/>
+<span class="annotation" title="Scope: {it.scope.name}">@{it.scope.localName}</span><br/>
+{#for q in it.qualifiers}
+ {#if q.name != 'javax.inject.Named' && q.name != 'javax.enterprise.inject.Any' && q.name != 'javax.enterprise.inject.Default'}
+   <span class="annotation" title="Qualifier: {q.name}">@{q.localName}</span><br/>
  {/if}
 {/for}
-<span title="{it.type.name}">{it.type.localName}</span>
+<span title="{it.type.name}"><span class="package-name">{it.type.shortenedPackageName}.</span>{it.type.localName}</span>
```

Comment:
Frankly, I'm against shortening packages. You end up having absolutely no idea of in which area is the class and that can be a real pain.

Just have a look at your examples: `i.q.d.Foo`. If you class name is a bit generic, you end up guessing what it could be.

Reply:
(and btw, that's not a theoretical issue, it can be a pain sometimes to understand from where a log is coming given we do that for loggers)

Reply:
So first of all we would display `i.q.dummy.Foo` in dev UI because we use `getShortenedPackageName()` + the simple name. Furthermore, we set the FQCN in the title so if you wait a little with the pointer on the name the FQCN is shown. Finally, I believe that `i.q.dummy.Foo` is much better than `Foo` which is the current state.

That said, if you really think it's a big issue then I can remove the package info completely. Although I believe that it won't help ;-).

Reply:
Yeah, I would keep the FQCN.

Reply:
> Yeah, I would keep the FQCN.

Hehe, not sure what to keep here as the simple name is currently used ;-). The problem with FQCN is that long packages are usually not very readable. Anyway, I'll try to use the FQCN and we'll see how it goes...

Reply:
Done.